src/commands/workflow/run.ts

```diff
@@ -14,16 +14,15 @@ export default class WorkflowRun extends BaseCommand<typeof WorkflowRun> {
       default: "development",
       summary: "The environment in which to run the workflow",
     }),
-    recipients: Flags.string({
+    recipients: commaSeparatedJsonStr({
```

Comment:
Continuing from [this discussion](https://github.com/knocklabs/control/pull/2218#discussion_r1253163500), I think here we want to be able to support:
1. a comma separated string of one or more user ids (e.g. `--recipients=thomas`, or `--recipients=thomas,franco`)
2. a json parseable string that can have one or more of user ids and/or object reference maps (e.g. `--recipients='{"id": "foo", "collection": "bar"}'`, or `--recipients='[{"id": "one", "collection": "foo"},{"id": "two", "collection": "bar"}]'`, or `--recipients='["thomas",{"id": "two", "collection": "bar"}]'`)

And the easiest way to parse this would be, try `JSON.parse()`, if successful use that; otherwise fall back to using the given string as is and treat it as a user id.

Reply:
Gotcha! I will improve the implementation to support this.

Reply:
Here are the changes in order to support this: https://github.com/knocklabs/knock-cli/pull/190/files